src/App.test.js

```diff
@@ -1,8 +1,98 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render, screen, fireEvent } from "@testing-library/react";
+import Counter from "./components/Counter";
+import SearchForm from "./components/SearchForm";
+import GenreSelect from "./components/GenreSelect";
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe("Counter", () => {
+  it("Test that component renders initial value provided in props", () => {
```

Comment:
better to start all tests with `should` if possible, easier to understand what it's doing
```suggestion
  it("should render initial value provided in props", () => {
```